readme.md

```diff
@@ -1,7 +1,7 @@
 # Stars block
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://lishvets.github.io/layout_stars/)
```

Comment:
<img width="1482" alt="Screenshot 2023-08-07 at 16 30 07" src="https://github.com/mate-academy/layout_stars/assets/53228259/983b3089-a7d9-4a71-904c-dee7d7afaacc">

in master brach you had 2 icons, use them instead of png

Reply:
<img width="1325" alt="Screenshot 2023-08-07 at 16 31 06" src="https://github.com/mate-academy/layout_stars/assets/53228259/9dd9583a-5a39-4c7c-af2b-5fb2a539305c">

and delete all this images